apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ErrorReportingExtension.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using Google.Api.Gax;
+using Google.Cloud.ErrorReporting.V1Beta1;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    /// <summary>
+    /// Extensions for Error Reporting.
+    /// </summary>
+    public static class ErrorReportingExtension
```

Comment:
Marked internal.

I wasn't aware of a better way to to do this.  It seems silly to convert to obj -> json then to json -> struct.  It would be nice to do obj -> struct without having to manually convert.